src/rest_api/general_application.py

```diff
@@ -302,42 +241,39 @@ def post(self) -> object:
                 .format(video_path)
             )
 
-        _thread.start_new_thread(
-            process_data_and_extract_emotions_with_song,
-            (
-                video_id,
-                video_path,
-                video_time_range,
-                song_id
-            )
-        )
+        # Send request to the video API
 
         return {'Response': 'The request has been sent and should be'
```

Comment:
Gotta love splunk

Reply:
This can be removed when the actual endpoints are being finished :)